### PR TITLE
LPS-76030

### PR DESCRIPTION
**Pull request by noahsherrillwork**
/cc @drewbrokke, @willnewbury, @JorgeFerrer

### CODE REVIEW

**Comment by noahsherrillwork** [comment]
ci:test

**Comment by noahsherrillwork** [comment]
ci:reopen

File: modules/apps/foundation/user-associated-data/user-associated-data-test-util/src/main/java/com/liferay/user/associated/data/test/util/WhenHasStatusByUserIdField.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+/**
+ * @author Noah Sherrill
+ */
+public interface WhenHasStatusByUserIdField {
+
+	public void addDataObjectWithStatusByUserId(
+			long userId, long statusByUserId)
+		throws Exception;
+
+}
```
Comment (drewbrokke) [inline modules/apps/foundation/user-associated-data/user-associated-data-test-util/src/main/java/com/liferay/user/associated/data/test/util/WhenHasStatusByUserIdField.java]: Would this class work as an inner class of the base test case?

**Comment by drewbrokke** [comment]
@noahsherrillwork I'd say let's create a unique ticket for this as well since it's not specific to Bookmarks.

File: modules/apps/foundation/user-associated-data/user-associated-data-test-util/src/main/java/com/liferay/user/associated/data/test/util/WhenHasStatusByUserIdField.java (hunk 1 above)
Comment (noahsherrillwork) [reply modules/apps/foundation/user-associated-data/user-associated-data-test-util/src/main/java/com/liferay/user/associated/data/test/util/WhenHasStatusByUserIdField.java]: I'm not sure whether that could work. Some subclasses of the base test case will implement this interface, as BookmarksEntryUADEntityAggregatorTest does. It doesn't seem like a typical use case of an inner class. Also, the pattern used here was adopted from BaseTrashHandlerTestCase.

Comment (drewbrokke) [reply modules/apps/foundation/user-associated-data/user-associated-data-test-util/src/main/java/com/liferay/user/associated/data/test/util/WhenHasStatusByUserIdField.java]: Got it.  Thanks for explaining.

**Pull request closed without merging**